paymentsheet/src/main/java/com/stripe/android/paymentsheet/viewmodels/BaseSheetViewModel.kt

```diff
@@ -53,9 +52,6 @@ internal abstract class BaseSheetViewModel(
     val editInteractorFactory: ModifiableEditPaymentMethodViewInteractor.Factory,
     val isCompleteFlow: Boolean,
 ) : AndroidViewModel(application) {
-    internal val googlePayState: StateFlow<GooglePayState> = savedStateHandle
-        .getStateFlow(SAVE_GOOGLE_PAY_STATE, GooglePayState.Indeterminate)
-
```

Comment:
Just to validate, we were never using the Google Pay state to determine anything after process death right?

Reply:
paymentMethodMetadata is also restored on process death. So we should be safe!